tests/unit_tests/ensemble_evaluator/test_ensemble_legacy.py

```diff
@@ -99,7 +89,11 @@ def _test_run_and_cancel_legacy_ensemble(tmpdir, make_ensemble_builder, monkeypa
 
 
 @pytest.mark.timeout(10)
-def test_run_legacy_ensemble_exception(tmpdir, make_ensemble_builder, monkeypatch):
+def test_run_legacy_ensemble_with_bare_exception(
+    tmpdir, make_ensemble_builder, monkeypatch
+):
+    """This test function is not ported to Scheduler, as it will not
+    catch general exceptions."""
```

Comment:
The logs have been searched through, and the general exception that the legacy queue can catch has not happened the last 3 months.